### ISSUE TITLE
Single Powerline Extender doesn't allow long enough wires

### ISSUE THREAD

**Issue by jayzosayers** [DavidHGillen/Satisfactory_AB-CableChoices#11]
The single powerline extender doesn't appear to be recognised as a transmission line as when you try to run wires between them their maximum length is significantly shorter than the regular transmission towers. If you look at my included screenshot you can see it maxes out around the same length as the regular powerlines. The other two extension pieces with two wires are fine.
![Uploading Satisfactory   11_07_2024 00_33_23.png…]()

**Comment by DavidHGillen** [comment]
You're right, somehow missed this in my testing. I will have to figure out how to get this working reliably as I just tested the obvious solution and it didn't take.

**Comment by jayzosayers** [comment]
A workaround for the time being is to use the heavy wires included in the mod because they can run further, but doing that also allows the transmission towers to be l

**Comment by DavidHGillen** [comment]
Did a refresh of the Extenders where I gave them all normal connectors. This allowed me to solve the issue without breaking saves and give the single points real long span connectors. Closing for now as this should solve the issue.
